src/lightning/data/streaming/cache.py

```diff
@@ -48,10 +59,20 @@ def __init__(
         super().__init__()
         if not _TORCH_GREATER_EQUAL_2_1_0:
             raise ModuleNotFoundError("PyTorch version 2.1 or higher is required to use the cache.")
+
```

Comment:
Shouldn't we cover the newly added logic in a test?

Reply:
done !